docs/learn/learn-accounts.md

```diff
@@ -112,13 +111,36 @@ For instance, `//bill//account//1` and `//john/polkadot/initial` are both valid.
 derived account, you must know both the seed and the derivation path, so you should either use a
 well-defined sequence (e.g. //0, //1, //2...) or be sure to write down any derivation paths you use.
 
-
-:::info 
+:::info
 
 It is not possible to generate a derived account without also knowing the derivation path.
 
 :::
 
+There is an additional type of derivation called password derivation. On Polkadot you can derive a
+**password key** account using **///** after the mnemonic phrase
+
+```
+'caution juice atom organ advance problem want pledge someone senior holiday very///0'
+```
+
+In this type of derivation, if the seed would leak, accounts cannot be derived without the initial
+password. In fact, if one knows the seed and the soft or hard derivation path, will have access to
+your account because the account password is only used to sign transactions on a specific device.
```

Comment:
That line is confusing - you are referring to the account password set on the extension or the device etc, which can be mistaken for the password in the password derived accounts. Might want to rephrase or remove completely. It is explained the above section that knowing the mnemonic phrase and the derviation path will give full access to the derived account. 

```suggestion
your account.
```